Mark deprecated all mentions of authmanager and webconfig (#199)
cossacklabs/product-docs: acra/configuring-maintaining/general-configuration/acra-server.md

AcraServer handles HTTP requests that may change its internal state, generates new Zones or fetches data
related with authentication of AcraWebConfig users.

{{< hint warning >}}

> AcraWebConfig and AcraAuthManager are deprecated and will not be available after 0.90.0.

Reviewer: ```suggestion
AcraWebConfig and AcraAuthManager are deprecated and will not be available since 0.91.0.
```

  incorrect request
  ```

> - Endpoint: `/loadAuthData` (deprecated after 0.90.0).

Reviewer: ```suggestion
- Endpoint: `/loadAuthData` (deprecated since 0.91.0).
```

  incorrect request
  ```

> - Endpoint: `/getConfig` (deprecated after 0.90.0).

Reviewer: ```suggestion
- Endpoint: `/getConfig` (deprecated since 0.91.0).
```

  incorrect request
  ```

> - Endpoint: `/setConfig` (deprecated after 0.90.0).

Reviewer: ```suggestion
- Endpoint: `/setConfig` (deprecated since 0.91.0).
```
